src/Tools/SeleniumGenerator/Helpers/RoslynHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace DotVVM.Framework.Tools.SeleniumGenerator.Helpers
+{
+    public static class RoslynHelper
+    {
+        public static ObjectCreationExpressionSyntax GetPathSelectorObjectInitialization(string propertyName)
+        {
+            return SyntaxFactory.ObjectCreationExpression(
```

Comment:
I don't think we'd lose anything if we just used string concatenation instead of Roslyn AST... it does not solve any problem for us (not even correct parenthesis) and is super annoying to use.